src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/PassingTimesEditor/validateForm.ts

```diff
@@ -31,15 +34,23 @@ const hasAtleastOneFieldSet = (passingTime: any) => {
 };
 
 export const validateTimes = (
-  passingTimes: any[],
-  intlState: any
+  stopPoints: StopPoint[],
+  passingTimes: PassingTime[],
+  intlState: IntlState
 ): { isValid: boolean; errorMessage: string } => {
-  if (!(passingTimes?.length >= 2))
+  const intl = getIntl({ intl: intlState });
+
+  if (stopPoints.length < 2)
+    return {
+      isValid: false,
+      errorMessage: intl.formatMessage(messages.stopPointsInfo)
+    };
+
+  if (stopPoints.length > passingTimes.length)
```

Comment:
Kan detta ske?

Reply:
Ja, dessverre. Er nok en liten bug at det kan skje, så noe vi kan fikse. For nå må vi bare få inn endringen ti demo :)